cmd/skopeo/manifest.go

```diff
@@ -11,17 +11,23 @@ import (
 )
 
 type manifestDigestOptions struct {
+	deprecatedTLSVerify *deprecatedTLSVerifyOption
 }
 
 func manifestDigestCmd() *cobra.Command {
-	var opts manifestDigestOptions
+	deprecatedTLSVerifyFlags, deprecatedTLSVerifyOpt := deprecatedTLSVerifyFlags()
+	opts := manifestDigestOptions{
+		deprecatedTLSVerify: deprecatedTLSVerifyOpt,
```

Comment:
BTW it makes so little sense to accept `skopeo manifest-digest --tls-verify` and the like, that I’m very tempted to not add these compatibility-only flags and just break those users.

Reply:
I don't think it was ever documented in the `manifest` man page. Nor `copy` nor `sync`, where it also makes no sense. I see no reason to support it in any of those.

Reply:
I think we did have `skopeo copy --tls-verify=false` (meaning `--src-tls-verify=false && --dest-tls-verify=false`) for a time. That’s one of the reasons why the global option is deprecated, to nudge users towards only disabling TLS for one of the two registries instead of both.

But I’ll remove it from the non-registry users.